vars/gradleCheckFlakyTestDetector.groovy

```diff
@@ -30,14 +30,14 @@ void call(Map args = [:]) {
             def awsSecretKey = env.AWS_SECRET_ACCESS_KEY
             def awsSessionToken = env.AWS_SESSION_TOKEN
             def timeFrame = args.timeFrame ?: '30d'
-            def indexName = 'gradle-check-*'
+            def indexName = 'gradle-check'
             def postMergeFailedTests = new FetchPostMergeFailedTestClass(metricsUrl, awsAccessKey, awsSecretKey, awsSessionToken, indexName, this).getPostMergeFailedTestClass(timeFrame)
             postMergeFailedTests.each { failedTest ->
                 def testData = []
                 def allPullRequests = []
                 def postMergeTestGitReference = new FetchPostMergeTestGitReference(metricsUrl, awsAccessKey, awsSecretKey, awsSessionToken, indexName, this).getPostMergeTestGitReference(failedTest)
                 postMergeTestGitReference.each { gitReference ->
-                    def failedTestNames = new FetchPostMergeFailedTestName(metricsUrl, awsAccessKey, awsSecretKey, indexName, awsSessionToken, this).getPostMergeFailedTestName(failedTest, gitReference)
```

Comment:
This is the actual fix right, for the above `Fetch*` classes its only adding the catch block.

Reply:
Main fix using alias `gradle-check` to fetch records instead of index-pattern `gradle-check-*`. This fix broke as the parameters are incorrectly passed. Fixed both the issues.